Formula/qt@5.9.rb

```diff
@@ -0,0 +1,143 @@
+# Qt 5.9 LTS is kept as a versioned formula because it's an LTS release, under support
+# until May 2020.
+# See https://github.com/Homebrew/homebrew-core/issues/35883
+#
+# Patches for Qt must be at the very least submitted to Qt's Gerrit codereview
+# rather than their bug-report Jira. The latter is rarely reviewed by Qt.
+class QtAT59 < Formula
+  desc "Cross-platform application and UI framework, Long-Term Support version"
+  homepage "https://www.qt.io/"
+  url "https://download.qt.io/official_releases/qt/5.9/5.9.3/single/qt-everywhere-opensource-src-5.9.3.tar.xz"
+  mirror "https://www.mirrorservice.org/sites/download.qt-project.org/official_releases/qt/5.9/5.9.3/single/qt-everywhere-opensource-src-5.9.3.tar.xz"
+  sha256 "57acd8f03f830c2d7dc29fbe28aaa96781b2b9bdddce94196e6761a0f88c6046"
+
+  keg_only :versioned_formula
+
+  option "with-docs", "Build documentation"
+  option "with-examples", "Build examples"
+
+  depends_on "pkg-config" => :build
+  depends_on :xcode => :build
+  depends_on "mysql" => :optional
+  depends_on "postgresql" => :optional
+
+  # Restore `.pc` files for framework-based build of Qt 5 on OS X. This
+  # partially reverts <https://codereview.qt-project.org/#/c/140954/> merged
+  # between the 5.5.1 and 5.6.0 releases. (Remove this as soon as feasible!)
+  #
+  # Core formulae known to fail without this patch (as of 2016-10-15):
+  #   * gnuplot  (with `--with-qt` option)
+  #   * mkvtoolnix (with `--with-qt` option, silent build failure)
+  #   * poppler    (with `--with-qt` option)
+  patch do
```

Comment:
What's the scope on either of these patches being merged upstream?

Reply:
The first patch... is a long answer. I've spent days in the weeds on this with Octave recently and I still don't know. (That is, I'm still not clear on what the exact issue is, and whether it's landed already.)

The second patch got landed in Qt 5.10; I don't think it'll be backported to the 5.9 series.

Reply:
@apjanke I think we should aim to include this without these patches unless it's useless without. At the very least would be good to request/attempt a backport for these if it's an LTS.

Reply:
First patch was landed in 5.12. I'm not aware of a bug report for it however.

Reply:
Backports for this probably require using their Gerrit (https://codereview.qt-project.org). Would like to see them requested before this is merged.